### PR TITLE
Fix typos in WAL arguments doc

### PR DESCRIPTION
**Pull request by codesome**

### CODE REVIEW

File: docs/configuration/arguments.md [hunk 1]
```diff
@@ -308,7 +308,7 @@ It also talks to a KVStore and has it's own copies of the same flags used by the
 #### WAL
 
 - `--ingester.wal-dir`
```
Comment (pracucci) [inline docs/configuration/arguments.md]: I've just noticed that the WAL flags are documented with the `--` prefix while it should be a single `-`. May you fix it, please?

Comment (codesome) [reply docs/configuration/arguments.md]: Will fix it tomorrow

**Comment by codesome** [comment]
Flags fixed

**Comment by pracucci** [comment]
Thanks @codesome !

**Pull request merged**